### PR TITLE
Updating: adding infra feature operatorhub test

### PR DESCRIPTION
**Pull request by gallettilance**

### CODE REVIEW

**Review by camilamacedo86 — APPROVED** [review]
/lgtm 
👍

**Comment by dinhxuanvu** [comment]
/hold
For further review on OpenShift-specific part.

File: pkg/validation/internal/operatorhub.go [hunk 1]
```diff
@@ -69,6 +69,12 @@ var validCategories = map[string]struct{}{
 	"Streaming & Messaging":  {},
 }
 
+var validInfraFeatures = map[string]struct{}{
+	"Disconnected": {},
```
Comment (camilamacedo86) [inline pkg/validation/internal/operatorhub.go:73]: @gallettilance based on the latest email in pm sent by @jmccormick2001 then, we might also allow disconnected too. Also, would that be valid for the others options as well. WDYT @jmccormick2001?

Comment (jmccormick2001) [reply pkg/validation/internal/operatorhub.go:73]: correct, my understanding is that all the values specified in the current docs (all lowercase) are valid within console and that the older style of spellings are also acceptable for now.

File: pkg/validation/internal/operatorhub.go [hunk 2]
```diff
@@ -69,6 +69,12 @@ var validCategories = map[string]struct{}{
 	"Streaming & Messaging":  {},
```
Comment (camilamacedo86) [inline pkg/validation/internal/operatorhub.go:69]: @gallettilance is missing only rebase. Otherwise, it seems /LGTM for me.

**Comment by perdasilva** [comment]
closing as stale

**Pull request closed without merging**